extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/RedPandaKafkaContainer.java

```diff
@@ -57,6 +57,7 @@ protected void containerIsStarting(InspectContainerResponse containerInfo, boole
         command += "--memory 1G --overprovisioned --reserve-memory 0M ";
         command += String.format("--kafka-addr %s ", getKafkaAddresses());
         command += String.format("--advertise-kafka-addr %s ", getKafkaAdvertisedAddresses());
+        command += "--set redpanda.auto_create_topics_enabled=true ";
```

Comment:
Seriously! Wow... that's a big change of behavior!

Reply:
I edited the description because I couldn't make our tests work with v22.2.2. I get NotLeaderOrFollowerException from record send. I suspect producer metadata are not in sync after topic creation.